apps/wallet/src/dapp-interface/WalletStandardInterface.ts

```diff
@@ -169,9 +169,10 @@ export class SuiWallet implements Wallet {
 
 	#setAccounts(accounts: GetAccountResponse['accounts']) {
 		this.#accounts = accounts.map(
-			({ address, publicKey }) =>
+			({ address, publicKey, nickname }) =>
 				new ReadonlyWalletAccount({
 					address,
+					label: nickname || undefined,
```

Comment:
I might or might not have known until just now that there was a `label` field on the wallet standard account 😆

Reply:
I'm kind of okay exposing this but I do have a tiny concern that folks might not expect dapps to read nicknames. But that's probably paranoia.